postgres/postgresql.conf.erb

```diff
@@ -33,13 +32,20 @@ lc_numeric = 'en_AU.UTF-8'
 lc_time = 'en_AU.UTF-8'
 default_text_search_config = 'pg_catalog.english'
 
+idle_in_transaction_session_timeout = '120s'
+
 # These settings only apply to the master.
-wal_level = hot_standby
+wal_level = replica
```

Comment:
I don't think I know enough about replication to give you an answer. I'll do some reading and get back to you 😜